### ISSUE TITLE
strip_tags() should be used when rendering get_the_title() as a title attribute.

### ISSUE THREAD

**Issue by mfields** [Automattic/_s#184]
Failure to strip the tags included in `get_the_title()` allows html tags to be included in title attributes. I have no idea how a screen reader will deal with these, but I have a feeling that it will add unneeded noise to those listening to the title attributes. It adds quite a bit of visual noise as well:

![HTML code in the title attribute](http://download.mfields.org/screenshot/2013-03-18_2153.png)

This also happens when hovering over the image too:

![HTML code in the title attribute](http://download.mfields.org/screenshot/2013-03-18_2209.png)
